tests/llmAnalytics.spec.ts

```
@@ -0,0 +1,31 @@
+const languagesToTestLlm = ["ar", "en"];
+const requestLlmBody = getRequestBody("llm-request");
+const [llmAnalyticsUrl] = getApis(["llmAnalytics"], "llm-analytics").map(
+  (apiObj) => apiObj?.url
+);
+
+languagesToTestLlm.forEach((languageToTestLlm) => {
+  test(`Check ${llmAnalyticsUrl} to give correct language response (${languageToTestLlm})`, async ({
+    request,
+  }) => {
+    expect(llmAnalyticsUrl).toBeDefined();
+    const response = await request.post(llmAnalyticsUrl!, {
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)